src/commonMain/kotlin/exchange.dydx.abacus/state/manager/V4StateManagerAdaptor.kt

```diff
@@ -1195,7 +1195,10 @@ class V4StateManagerAdaptor(
             submitPlaceOrder(callback, it, analyticsPayload, true)
```

Comment:
might we need to remove the callback here and line 1186?

Reply:
hmm yeah so that's accurate - right now there's a callback for _every_ cancel and place order message. `submitPlaceOrder` is currently a shared function between the trade form flows and trigger form flows. If we wanted to only send back a single callback for the aggregate order messages we'd have to create a new function that only calls `send` once - though I'm unsure about the details of when to send `error` vs no error (what if one transaction fails but the other passes)?